stories/examples/pca.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, Plot, Heading, ScatterSeries, Annotations } from '../../src';
+import { Ellipse } from '../../src/components/Annotations/Ellipse';
+import { Text } from '../../src/components/Annotations/Text';
+import data from '../data/pca.json';
+import { DEFAULT_PLOT_CONFIG } from '../utils';
+
+export default {
+  title: 'Examples/PCA example',
+} as Meta;
+interface Point {
+  x: number;
+  y: number;
+  color: {
+    shape: string;
+    fill: string;
+    r: number;
+  };
+  id: string;
+}
+export function PCAExample() {
+  return (
+    <Plot {...DEFAULT_PLOT_CONFIG}>
+      <Heading title="COVID-19 cases in USA (2020)" />
+      <ScatterSeries
+        markerStyle={{
+          fill: (point: Point) => point.color.fill,
+          stroke: 'none',
```

Comment:
Is this needed?

Reply:
Yes because in default points have black strokes